content/articles/2018-02-rlang-0.2.0.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: rlang 0.2.0
+slug: rlang-0.2.0
+description: >
+  rlang 0.2.0 is on CRAN! This version brings many improvements to
+  tidy evaluation.
+author: Lionel Henry
+date: 2018-02-26
+categories: [package]
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+library("rlang")
+options(crayon.enabled = TRUE)
+```
+
+The r-lib team is happy to announce a new version of rlang that
```

Comment:
For some reason, saying "r-lib team" sounds wrong to me, even though "tidyverse team" seems ok. Maybe just "we"?

Reply:
First use of package name should link to package website